src/features/wallet/scan/ScanView.tsx

```diff
@@ -122,7 +123,7 @@ const ScanView = () => {
               fontSize={18}
               marginBottom="s"
             >
-              Find a QR Code to scan
+              {t('send.scan.title')}
```

Comment:
you did the externalizing too 😍

Reply:
the best junior react engineer

Reply:
in addition to `import { useTranslation } from 'react-i18next'` you'll need to define `t`. At the top of this component, add `const { t } = useTranslation()`